templates/account/payments.php

```diff
@@ -10,19 +10,19 @@
 		exit;
 	}
 
-	/**
-	 * @var array $VARS
-	 * @var Freemius $fs
-	 */
-	$fs = freemius( $VARS['id'] );
+    /**
+     * @var array $VARS
+     * @var Freemius $fs
+     */
+    $fs = freemius( $VARS['id'] );
 
-	$slug = $fs->get_slug();
+    /**
+     * @var FS_Payment[] $payments
+     */
+    $payments = freemius( $VARS['payments'] );
```

Comment:
@msladu I just noticed you wrapped `$VARS['payments']` within `freemius()`, this is a mistake, it should simply be equal to `$VARS['payments']`.

Reply:
@vovafeldman Will fix it.